Automattic/wp-calypso
# Add map_block_map_provider global variable

## Proposed Changes

This PR injects a `map_block_map_provider` variable containing the output of the `wpcom_map_block_map_provider` filter into the `window` object.


## Testing Instructions

1. Follow the directions to sync ETK to your sandbox: PCYsg-ly5-p2
2. Add a new post/page
3. Open devtools & make sure the iframe (post-new.php) is selected 
4. Output `map_block_map_provider`

![CleanShot 2023-04-19 at 17 19 56@2x](https://user-images.githubusercontent.com/528287/233123125-d2b6c81b-fc01-46cd-a9f3-c3a2dfaa25e8.png)


## Pre-merge Checklist
